features/support/env.el

```diff
@@ -21,6 +21,10 @@
 (defvar ert-runner-output "")
 (defvar ert-runner-error "")
 
+(require 'undercover)
+(undercover "*.el"
+	    (:exclude "*-test.el")
+	    (:report-file "/tmp/undercover-report.json"))
```

Comment:
No need in this specific setting indeed.
I just reused the template from cask-package-toolset.

This is needed when you want to merge report from ert and ecukes